src/components/panel/panel.spec.js

```diff
@@ -1053,6 +1053,125 @@ describe('$mdPanel', function() {
     });
   });
 
+  describe('grouping logic:', function() {
```

Comment:
Why is there a `:` in the describe? - This does not follow the normal jasmine pattern.